python/ray/tune/tuner.py

```diff
@@ -42,14 +42,6 @@
 _SELF = "self"
 
 
-_TUNER_FAILED_MSG = (
```

Comment:
Why we no longer send this failure message?

Reply:
This only caught `TuneError`, which only gets raised from `tune.run(raise_on_failed_trial=True)`. However, the Tuner always passes `False` for this flag, so this error message doesn't actually get printed ever.

https://github.com/ray-project/ray/blob/93d8d962102a75a505acfa1c26346676bd2fd023/python/ray/tune/impl/tuner_internal.py#L594